worker/label/label.go

```diff
@@ -9,6 +9,7 @@ const (
 	Hostname            = prefix + "hostname"
 	Network             = prefix + "network" // "cni" or "host"
 	ApparmorProfile     = prefix + "apparmor.profile"
+	SELinux             = prefix + "selinux"              // "true" or "false"
```

Comment:
Could be "enabled" and  "disabled"

```suggestion
	SELinux             = prefix + "selinux"              // SELinux support ("enabled", "disabled")
```

Reply:
@tonistiigi WDYT?

Reply:
No strict opinion. Personally would prefer values that work with `ParseBool`. Or `selinux.enabled` maybe in that case?

Reply:
Changed to `selinux.enabled`